frontend/packages/dev-console/src/components/edit-application/EditApplicationForm.tsx

```diff
@@ -56,10 +56,12 @@ const EditApplicationForm: React.FC<FormikProps<FormikValues> & EditApplicationF
         {createFlowType === CreateApplicationFlow.Container && <ImageSearchSection />}
         {createFlowType === CreateApplicationFlow.Container && <IconSection />}
         <AppSection project={values.project} />
-        <PipelineSection
-          builderImages={builderImages}
-          existingPipeline={appResources?.pipeline?.data}
-        />
+        {createFlowType !== CreateApplicationFlow.Container && (
```

Comment:
We need tests to make sure the expected items are what we expect.

This may be one of those few cases where we test static (as well as conditional) renderings.

Reply:
Added tests.